server/server_test.go

```diff
@@ -58,6 +60,30 @@ func TestMakeURLToEndpoint(t *testing.T) {
 	)
 }
 
+func TestAddCORSHeaders(t *testing.T) {
+	mockStorage := helpers.MustGetMockStorage(t, true)
+	defer helpers.MustCloseStorage(t, mockStorage)
+
+	err := mockStorage.WriteReportForCluster(testdata.OrgID, testdata.ClusterName, "{}", time.Now())
+	if err != nil {
```

Comment:
Would be slightly better to use `helpers.FailOnError(t, err)`

Reply:
Yes, could be.